changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+
+## v0.13.3 | tbd
+
+#### New features
+* Added `lk.colormaps` with custom colormaps for plotting single-channel images. Note: this is a `namedtuple` so you can access the attributes using the dot notation (for example `kymo.plot("blue", cmap=lk.colormaps.cyan)`). The available attributes are `red`, `green`, `blue`, `magenta`, `yellow`, `cyan`.
```

Comment:
agreed about the docs, it's just kind of awkward because there's no specific place to put this description. So for now, I ended up putting a blurb alongside the documentation on the plotting methods for `Scan` `Kymo` and `CorrelatedStack`. I kept the changelog the same since there's not really a convenient place in the docs to link to

Reply:
Where is the blurb?